tools/convert_models/publish_model.py

```diff
@@ -21,18 +23,27 @@ def process_checkpoint(in_file, out_file):
         del checkpoint['optimizer']
     # if it is necessary to remove some sensitive data in checkpoint['meta'],
     # add the code here.
-    torch.save(checkpoint, out_file)
+    if torch.__version__ >= '1.6':
```

Comment:
Considering to use `mmcv.digit_version` to check version.

Reply:
Done